lib/screens/create_region_screens/select_region/select_region.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/components/flat_button_long.dart';
+import 'package:seeds/utils/build_context_extension.dart';
+
+class SelectRegion extends StatelessWidget {
+  const SelectRegion({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: Text(context.loc.createRegionSelectRegionAppBarTitle),
+        ),
+        bottomNavigationBar: SafeArea(
+          minimum: const EdgeInsets.all(16),
+          child: FlatButtonLong(title: "${context.loc.createRegionSelectRegionButtonTitle} (1/5)", onPressed: () {}),
```

Comment:
1/5 will probably need to be 4/5? 

Also, i know you dont have blocks or states yet, but i think that value will eventually go in the state?

Reply:
Yes this one will we actually 5/5. Not sure yet if it would be in the state.